AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/CampaignManagerInviteController.cs

```diff
@@ -68,15 +70,33 @@ public CampaignManagerInviteController(IMediator mediator, UserManager<Applicati
                 }
 
                 var userToInvite = await _userManager.FindByEmailAsync(invite.InviteeEmailAddress);
-                if (userToInvite != null && userToInvite.ManagedCampaigns.Any(m => m.CampaignId == campaignId))
+                if (userToInvite?.ManagedCampaigns != null && userToInvite.ManagedCampaigns.Any(m => m.CampaignId == campaignId))
                 {
                     ModelState.AddModelError("InviteeEmailAddress", $"{invite.InviteeEmailAddress} is allready a manager for this campaign");
                     return View("Send", invite);
                 }
 
                 invite.CampaignId = campaignId;
                 var user = await _userManager.GetUserAsync(User);
-                await _mediator.SendAsync(new CreateCampaignManagerInviteCommand { Invite = invite, UserId = user.Id });
+                var inviteId = await _mediator.SendAsync(new CreateCampaignManagerInviteCommand { Invite = invite, UserId = user.Id });
+
+                await _mediator.PublishAsync(new SendCampaignManagerInvite
```

Comment:
When publishing events, the event should be raised from where the action happened.  The event should be phrased as an event `SomethingHappened` and not a command `DoSomething`.  So just a couple of naming/location things to sort out.

As an example, if the event were named `CampaignManagerInvited` and raised from within the `CreateCampaignManagerInviteCommand` I think we'd be on track.